### PR TITLE
Support for Options of a nullable type

### PR DESCRIPTION
**Pull request by dwright-baneng**
Hello, Nice project.  Thanks for offering it.
I wanted to be able to have options that are nullable, e.g.:
```cs
	class Arguments
	{
		[Option("--integer", "An integer")]
		public int? Integer { get; set; }
	}
```

### CODE REVIEW

File: Jarilo.Tests/OptionsTests/Nullable/Tests.cs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        {
+            var args = $"{Command.Name} --nullable 47";
+            await AppTest.Run(args, output =>
+            {
+                output.AssertOption(47);
+            });
+        }
+
+        [Fact]
+        public async Task CommaSeparator_Success()
```
Comment (ninjah187) [inline Jarilo.Tests/OptionsTests/Nullable/Tests.cs]: `CommaSeparator_Success` and `DotSeparator_Success` tests are checking the same thing. Can you amend tests to remove this redundancy? Also `*Separator` is probably not the best naming as I can't see any separators check here.

**Comment by ninjah187** [comment]
Hey, thanks for your contribution! Please have a look at my comment regarding tests. Besides that all looks good to me and I'll be happy to merge your changes 😃

File: Jarilo.Tests/OptionsTests/Nullable/Tests.cs (hunk 1 above)
Comment (dwright-baneng) [reply Jarilo.Tests/OptionsTests/Nullable/Tests.cs]: I agree.  I had copied from OptionsTests/Integer without studying them close enough.

**Comment by ninjah187** [comment]
I just released on nuget version 1.2.0 containing your changes. Thanks and happy coding!

**Pull request merged**